.drone.yml

```diff
@@ -1,6 +1,6 @@
 workspace:
   base: /go
-  path: src/github.com/jpoles1/gopherbadger
+  path: src/github.com/dcaponi/gopherbadger
```

Comment:
This change should not be merged with the main branch.

Reply:
Totally understand, I haven't quite figured out a good workflow for submitting PRs to open source projects yet and I had to do this to get it to work on my machine. I'll changed these back to point to your project when I'm done.